pkg/util/kubernetes/pod.go

```diff
@@ -114,6 +114,8 @@ func natsPodMetricsContainer(image, tag, pullPolicy string) v1.Container {
 			"-routez",
 			"-subz",
 			"-varz",
+			"-channelz",
+			"-serverz",
```

Comment:
btw these two flags will only work for the monitoring endpoint of the streaming server which this operator does not support yet